### PR TITLE
FOCUS #55: Appendix section for Origination of cost data

### PR DESCRIPTION
**Pull request by udam-f2**
Appendix section for Origination of cost data

### CODE REVIEW

File: specification/appendix/origination_of_cost_data.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Origination of cost data
+
+Cost data presented in the billing datasets originates from various sources depending on the purchasing mechanism. There are at least 3 different pieces of information that are important for understanding where cost originated from.
+
+* Provider: The entity that made the infrastructure and/or services available for purchase.
+* Publisher: The entity that produced the infrastructure and/or services that were purchased
+* Invoicing Entity: The entity responsible for invoicing for the infrastructure and/or services consumed
```
Comment (flanakin) [inline specification/appendix/origination_of_cost_data.md:7]: "Entity" is generic. Is there another term that's more specific? Maybe "Invoice Issuer"?
```suggestion
* Invoicing Entity: The entity responsible for invoicing for the infrastructure and/or services consumed.
```

File: specification/appendix/origination_of_cost_data.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Origination of cost data
+
+Cost data presented in the billing datasets originates from various sources depending on the purchasing mechanism. There are at least 3 different pieces of information that are important for understanding where cost originated from.
+
+* Provider: The entity that made the infrastructure and/or services available for purchase.
+* Publisher: The entity that produced the infrastructure and/or services that were purchased
```
[the remark this answers is not in the document]
Comment (ijurica) [reply specification/appendix/origination_of_cost_data.md:6]: What about "resources and/or services"?

File: specification/appendix/origination_of_cost_data.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Origination of cost data
```
[the remark this answers is not in the document]
Comment (udam-f2) [reply specification/appendix/origination_of_cost_data.md:1]: Discussed in the squad. Couldn't come up with anything the group felt was a better fit either. Keeping as is and can revisit later if needed

File: specification/appendix/origination_of_cost_data.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (udam-f2) [reply specification/appendix/origination_of_cost_data.md:6]: updated

File: specification/appendix/origination_of_cost_data.md (hunk 1 above)
Comment (udam-f2) [reply specification/appendix/origination_of_cost_data.md]: Updated

**Review by mike-finopsorg — APPROVED** [review]

**Pull request merged**